courses/fundamentals_of_ada/labs/solar_system/06_subprograms/sol/main.adb.pp

```diff
@@ -0,0 +1,204 @@
+-----------------------------------------------------------------------
```

Comment:
This file looks like a prettyprint result - probably shouldn't be here

Reply:
Good catch